package.json

```diff
@@ -58,7 +58,6 @@
     "pouchdb-vhost": "^1.0.2",
     "pouchdb-wrappers": "^1.3.6",
     "raw-body": "^2.2.0",
-    "sanitize-filename": "^1.6.1",
```

Comment:
Yeah that's a good point. `new PouchDB('prn')` currently breaks in LevelDB-based PouchDBs, which is not express-pouchdb's fault. We could just throw an error on Windows. 🤷‍♂️